README.md

```diff
@@ -42,14 +44,13 @@ If you decide not to use an 'alternate id', your list of eligible employees will
 > A CSV is just a file of comma-separated values, with a newline between each line. It looks like this:  
 My GitHub ID is:,Email Address  
 danisyellis,dgellis@indeed.com  
-anexample,example@example.com  
-nerys,kira@bm.gov  
-captsisko,bsisko@starfleet.com  
+octocat,octocat@github.com  
+thisShouldError,notaname@fake.com 
```

Comment:
I believe example.com is specifically reserved for examples, whereas fake.com is a real domain. So this should probably be notaname@example.com

Reply:
fixed this while doing the merge commit (merged with Ingrid's README changes).
Unfortunately, doing the merge in GitHub's online editor didn't allow me to change the commit message to mention this extra change. Not a biggie, just somewhat annoying.